.github/workflows/causal_lm_cpp.yml

```
@@ -274,6 +274,48 @@ jobs:
           && call .\ov\setupvars.bat
           && python samples\python\greedy_causal_lm\lora.py .\TinyLlama\TinyLlama-1.1B-intermediate-step-1431k-3T\ adapter_model.safetensors "How to create a table with two columns, one of them has type float, another one has type int?"
 
+  cpp-greedy_causal_lm-Chatglm3-6b:
+    runs-on: ubuntu-20.04-16-cores
```

```suggestion
    runs-on: ubuntu-20.04-4-cores
```

Maybe this or `ubuntu-20.04-8-cores` would be enough